tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# SPDX-FileCopyrightText: 2024-present Mark Janse <mark.janse@health-ri.nl>
+#
```

Comment:
Please add least one happy flow unit test which illustrates the usage of the library